README.md

```diff
@@ -93,6 +93,12 @@ cmake --build .        # Build all default targets.
 To run the tests, make sure you clone https://github.com/ethereum/tests and point the environment variable
 `ETHEREUM_TEST_PATH` to that path.
 
+After building,
+```
+ctest -j4
```

Comment:
there is also a cmake compiler flag for this.
```FASTCTEST```
if defined then only test suites will be added as a ctest cases. otherwise every test case is added as ctest test case

Reply:
This I wonder too @winsvega